daemon/src/taker_cfd.rs

```diff
@@ -196,6 +198,26 @@ impl<O, M, W> Actor<O, M, W> {
             })?;
         Ok(())
     }
+
+    async fn handle_auto_rollover(&mut self) -> Result<()> {
+        tracing::debug!("Auto rollover");
```

Comment:
Imho we should do a _smart_ auto rollover: 
check if the CFD's expiry is within 23h, if so, we rollover for another 24h. That way we can run it more regular for a low cost, e.g. every X minutes instead of hourly.

Reply:
Note to future Daniel: `within 23h` will cover the CFD already being past attestation as well.